incubator/mariadb-cluster/values-production.yaml

```diff
@@ -0,0 +1,216 @@
+## Bitnami MariaDB image
+image: beltran/mariadb:latest
```

Comment:
I guess you set this image for testing purposes. Please remove it from PR

Reply:
yes, thanks. I need to integrate the changes in the nami module

Reply:
also need to adopt the new structure

```
image:
  registry: docker.io
  repository: bitnami/mariadb
  tag: 1.2.3
  pullPolicy: Always
```